scenarios/kubernetes_e2e.py

```diff
@@ -582,6 +601,13 @@ def create_parser():
         '--build', nargs='?', default=None, const='',
         help='Build kubernetes binaries if set, optionally specifying strategy')
     parser.add_argument(
+        '--use-shared-build', nargs='?', default=None, const='',
+        help='Use prebuilt kubernetes binaries if set, optionally specifying strategy')
+    parser.add_argument(
+        '--gcs-shared',
```

Comment:
bikeshed: maybe `--shared-build-location`?

Reply:
this flag can be used for other shared files and isn't actually the build location but where the "symlink" to the build is.